### PR TITLE
refactor: eliminate brightray::BrowserMainParts

### PR DESCRIPTION
**Pull request by miniak**
#### Description of Change
More effort to eliminate brightray.

#### Checklist
- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

#### Release Notes
Notes: no-notes

### CODE REVIEW

**Comment by miniak** [comment]
@ckerr fixed

File: atom/browser/atom_browser_main_parts.cc [hunk 1]
```diff
@@ -20,28 +25,61 @@
...
+#include "base/base_switches.h"
 #include "base/command_line.h"
-#include "base/threading/thread_task_runner_handle.h"
+#include "base/feature_list.h"
+#include "base/message_loop/message_loop.h"
+#include "base/path_service.h"
+#include "base/strings/string_number_conversions.h"
+#include "base/strings/utf_string_conversions.h"
+#include "brightray/browser/brightray_paths.h"
+#include "brightray/common/application_info.h"
```
Comment (deepak1556) [inline atom/browser/atom_browser_main_parts.cc]: Can you move `brightray/browser/brightray_paths.h` in this PR, this file was the last user under `brightray`.

Comment (miniak) [reply atom/browser/atom_browser_main_parts.cc]: @deepak1556 I was planning to do this in the last PR

Comment (miniak) [reply atom/browser/atom_browser_main_parts.cc]: @deepak1556 https://github.com/electron/electron/pull/15240/commits/9c188cbc81fddab9866d943bb58f1518038af0bc

**Pull request merged**